composer/algorithms/cutmix/cutmix.py

```diff
@@ -184,6 +184,8 @@ class CutMix(Algorithm):
             box such that each pixel has an equal probability of being mixed.
             If ``False``, defaults to the sampling used in the original
             paper implementation. Default: ``False``.
+        input_key (Any, optional): A key that indexes to the input from the batch.
+        target_key (Any, optional): A key that indexes to the target from the batch.
```

Comment:
Can you add support for the `get_fn` and `set_fn` as well?

Reply:
+1 on this, otherwise Lgtm. Nice two line change.

Reply:
ok added support for `get_fn` and `set_fn`. To avoid extra arguments to the user and extra parsing boilerplate for the algorithm writer, `input_key` and `target_key` optionally accept a sequence of getter, setter. All the parsing logic is pushed down to `batch_set` and `batch_get`. Lemme know what you think, @A-Jacobson